nautobot/core/views/generic.py

```diff
@@ -940,6 +940,9 @@ def alter_obj(self, obj, request, url_args, url_kwargs):
         # For example, a parent object can be defined given some parameter from the request URL.
         return obj
 
+    def _extra_post_save_action(self, obj, form):
```

Comment:
Nit: since this is intended for views to override, I'm not sure that the `_` prefix is necessary/appropriate.